### PR TITLE
GNOME 45 updates

### PR DESCRIPTION
**Pull request by skewty**

### CODE REVIEW

**Comment by skewty** [comment]
The GUI would certainly benefit from some more work. What do you think of the new menus like Caffeine extension does?

File: extension.js [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
- * enable is the entry point called by gnome-shell
- */
-// eslint-disable-next-line no-unused-vars
-function enable() {
-    Logger.info("enabling containers extension");
-    containersMenu = new ContainersMenu();
-    Logger.debug(containersMenu);
-    Main.panel.addToStatusArea("containers-menu", containersMenu);
-}
+export default class ContainersExtension {
```
Comment (rgolangh) [inline extension.js:20]: can you please revert the format changes from line 21 - 39?

Comment (rgolangh) [reply extension.js:20]: scratch tht I see you added a class

**Comment by baco** [comment]
Is this merging soon?

**Comment by skewty** [comment]
The work I did does the conversion to GNOME 45 so it works but the GUI seems somewhat broken still. I was hoping someone would take what I did and finish the GUI parts but I don't think anyone did anything with it.

**Pull request merged**